### PR TITLE
[MRG+1] Circle ci anaconda

### LINKED ISSUE #1003: Use anaconda on CircleCI
I've heard rumors that anaconda on CircleCI could speed up very much run of the examples. We should give it a try. MNE-Python is trying too, maybe we can draw inspiration from there work:
https://github.com/mne-tools/mne-python/pull/2849/files

It would be interesting to see a before/after account of execution time.

This would be a first step in solving https://github.com/nilearn/nilearn/issues/1000

### PR DESCRIPTION
**Pull request by aabadie**
Tries to fix #1003, but still doesn't work. Any help/tips greatly appreciated.

### CODE REVIEW

**Comment by aabadie** [comment]
@GaelVaroquaux : awesome ! it worked https://circleci.com/gh/aabadie/nilearn/276 :beers:

File: continuous_integration/circle_ci_test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!bin/bash
+source activate testenv
+make clean test test-coverage
```
Comment (GaelVaroquaux) [inline continuous_integration/circle_ci_test.sh]: I think that we can remove this file, as we are no longer using it, right?

**Comment by GaelVaroquaux** [comment]
Great job!

I don't really see a speed up. Maybe this can be improved by installing the MKL, with "conda install mkl", for instance on line 31 of circle.yml.

**Comment by aabadie** [comment]
> Great job!
Thanks
>
> I don't really see a speed up. Maybe this can be improved by installing the MKL, with "conda install mkl", for instance on line 31 of circle.yml
Indeed, that would be worth a try. There is already a speedup in the install step which is now only taking about 1 minute and a half (instead of more than 10).

File: continuous_integration/circle_ci_test.sh (hunk 1 above)
Comment (aabadie) [inline continuous_integration/circle_ci_test.sh]: > In continuous_integration/circle_ci_test.sh:
> I think that we can remove this file, as we are no longer using it, right ?
Ah yes, I forgot this one. Will do it when time permits. In the mean time, I'll make conda less verbose using --quiet option. We are getting closer !

**Comment by arthurmensch** [comment]
Anaconda 2.5 is mkl enabled by default
Le 1 mars 2016 19:30, "Alexandre Abadie" <notifications@github.com> a
écrit :

**Comment by aabadie** [comment]
> Anaconda 2.5 is mkl enabled by default
Yes, I saw that but our install script removes it.

**Comment by aabadie** [comment]
@GaelVaroquaux, jI pushed a last commit adressing your previous comments. I also keep the use of mkl, just to be sure it's really faster. The answer is : not that much : https://circleci.com/gh/aabadie/nilearn/279

**Comment by aabadie** [comment]
Comparing with the previous successful build (https://circleci.com/gh/aabadie/nilearn/276), the documentation generation step is 8 minutes faster with mkl.

**Comment by aabadie** [comment]
Comparing with a master build (before the skip test PR merge), it's now 23 minutes faster (23% faster).

**Comment by GaelVaroquaux** [comment]
That looks great: it's faster, and the code is as complex, or maybe a bit simpler.

I such that we merge and that, in a separate pull request, someone enables Python3 (and fixes the examples that don't work in Python3)

**Comment by aabadie** [comment]
@GaelVaroquaux, I pushed one last commit for making comments clearer in CI scripts.

**Comment by AlexandreAbraham** [comment]
Merging when CircleCI happy. Thanks a lot!

**Comment by aabadie** [comment]
Travis is failing but the issue is not related to this PR : memory profiling test error (again)

**Pull request merged**